### DISCUSSION TITLE
Budget alles in 1

### DISCUSSION THREAD

**Discussion by Dnavis1993** [fabianishere/udm-iptv#56] [Q&A]
I'm trying to setup the UDM-Pro with Budget alles in 1 in the Netherlands. But I'm having issues when I spin up the container I lose internet connection. IPTV starts working immediately but internet stops working. 

I Think something is wrong with the routing but cannot find the resolution to resolve the issue. 

Budget is using the following vlans(Seems to be KPN/Telfort settings)
Internet: DHCP VLAN 34
IPTV: VLAN 4

DIAG Output
=== Configuration ===
WAN Interface: eth8
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
WAN VLAN: 4 (dev iptv)
WAN DHCP options:
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false

=== IP Link and Route ===
29: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue qlen 1000
    inet 10.138.74.185/22 brd 10.138.75.255 scope global iptv
       valid_lft forever preferred_lft forever
10.138.72.0/22 dev iptv  src 10.138.74.185
213.75.112.0/21 via 10.138.72.1 dev iptv  metric 229

 #ip route list
10.138.72.0/22 dev iptv proto kernel scope link src 10.138.74.185
82.169.149.0/25 dev eth8.34 proto kernel scope link src 82.169.149.119
192.168.1.0/24 dev br0 proto kernel scope link src 192.168.1.1
192.168.2.0/24 dev br4 proto kernel scope link src 192.168.2.1
213.75.112.0/21 via 10.138.72.1 dev iptv metric 229

I have tried it with multiple DHCP options but not luck yet. Reading the other discussions I do see some familiar issue with Bell Fibe. where someone is using the DHCP options "-n -o" Should I give that a try or do you see another issue with my setup?

**Comment by fabianishere** [comment]
I see you are currently not using any DHCP options. By default, it will then request a default route from the DHCP server. Could you try the default value: `-O staticroutes -V IPTV_RG`. This will ask the DHCP server for static routes and will not replace your default route.

**Reply by Dnavis1993** [reply]
Thanks @fabianishere for the quick reply, This is indeed one of the options that I also tried but that also caused the same issue. I'm havn't setup any routes myself. But if I should, do you have an Idea of what I should add? This for mee is not 100% clear.

**Reply by fabianishere** [reply]
So first set the DHCP options to `-O staticroutes -V IPTV_RG` and restart the service. 
Then let's see what's causing your internet to stop working.

**Comment by doktergroef** [comment]
Hey, I’m thinking about switching from KPN to _Budget alles in 1_ and I was wondering if udm-iptv works with that provider as well. Can I just use the same settings I had with KPN? Anyone here with experience on this? Thanks!

**Reply by furanity** [reply]
For as far I can see the technical specifications for Budget Alles in 1 are the same as KPN:

- https://www.budgetthuis.nl/alles-in-1/docs/BA_Glasvezel_Specificaties_Instellingen.pdf
- https://kpn.com/w3/file?uuid=563993a1-e48a-485a-90a8-7389c5d6f5d4&owner=8de86c61-bdc8-4848-872e-e5b380e682fc&contentid=50183&mode=incontext

So looking at the specs, that seems like the KPN script should work for Budget as well...

**Reply by doktergroef** [reply]
Thanks! I'll try it asap.
